portable/GCC/ARM_CR5/port.c

```diff
@@ -148,6 +149,10 @@
     #define portTASK_RETURN_ADDRESS    prvTaskExitError
 #endif
 
+/* The space on the stack required to hold the FPU registers.  This is 16 64-bit
+registers, plus a 32-bit status register. */

```

Comment:
In the VFPv3-D16 SP mode, this could be 32 x 32-bit registers rather than 16 x 64-bit.

Reply:
Yes, this is correct. However this refers only to the view of the registers. It can be either 16 x 64-Bit or 32 x 32-Bit. Here, this shouldn't be really relevant, as in both cases the same size of memory is being allocated in the stack. But one could mention this in the comment. 

Should I update the comment? Let me know, if you thinks this necessary.

Reply:
@ChristosZosi Ohh right. Cortex-R5 is a 32-bit cpu, so the target ARM registers are always 32-bit anyway. Clarifying the comment would be great if you can.